src/index.html

```diff
@@ -4,9 +4,78 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
-    <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="stylesheet" href="../styles.css">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="month">
+      <div class="month__day-1  month--start-Tuesday">
+      </div>
+      <div class="month__day-2">
+      </div>
+      <div class="month__day-3">
+      </div>
+      <div class="month__day-4">
+      </div>
+      <div class="month__day-5">
+      </div>
+      <div class="month__day-6">
+      </div>
+      <div class="month__day-7">
+      </div>
+      <div class="month__day-8">
+      </div>
+      <div class="month__day-9">
+      </div>
+      <div class="month__day-10">
+      </div>
+      <div class="month__day-11">
+      </div>
+      <div class="month__day-12">
+      </div>
+      <div class="month__day-13">
+      </div>
+      <div class="month__day-14">
+      </div>
+      <div class="month__day-15">
+      </div>
+      <div class="month__day-16">
+      </div>
+      <div class="month__day-17">
+      </div>
+      <div class="month__day-18">
+      </div>
+      <div class="month__day-19">
+      </div>
+      <div class="month__day-20">
+      </div>
+      <div class="month__day-21">
+      </div>
+      <div class="month__day-22">
+      </div>
+      <div class="month__day-23">
+      </div>
+      <div class="month__day-24">
+      </div>
+      <div class="month__day-25">
+      </div>
+      <div class="month__day-26">
+      </div>
+      <div class="month__day-27">
+      </div>
+      <div class="month__day-28">
+      </div>
+      <div class="month__day-29">
+      </div>
+      <div class="month__day-30">
+      </div>
+      <div class="month__day-31">
+      </div>
+    </div>
   </body>
+
```

Comment:
you havn't inplemented this point from chacklist 

<img width="756" alt="image" src="https://user-images.githubusercontent.com/100297341/201072310-b82e0fb0-b8d9-43ae-8418-b15d7b858267.png">

Reply:
I'm confused on this. I can create the modifier 

@for $index from 28 through 31 {
  .month-length:nth-child(#{$index}) {
    display: none;
  }
}

I then put class="month-length" in the html on the first day. It doesn't work and doesn't make sense to me. Is the length then set as a variable?
Where am I going wrong?